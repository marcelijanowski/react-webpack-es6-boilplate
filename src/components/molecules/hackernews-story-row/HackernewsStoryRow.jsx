import React, { PureComponent } from 'react';
import './HackernewsStoryRow.scss';

class HackernewsStoryRow extends PureComponent {
  render() {
    return (<div></div>)
  }
}

export default HackernewsStoryRow;