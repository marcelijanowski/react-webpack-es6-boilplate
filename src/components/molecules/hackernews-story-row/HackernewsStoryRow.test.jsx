import React, { PureComponent } from 'react'
import renderer from 'react-test-renderer';

import HackernewsStoryRow from './HackernewsStoryRow';

describe('HackernewsStoryRow container', () => {
  it('renders a snapshot', () => {
    const tree = renderer.create(<HackernewsStoryRow/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
})