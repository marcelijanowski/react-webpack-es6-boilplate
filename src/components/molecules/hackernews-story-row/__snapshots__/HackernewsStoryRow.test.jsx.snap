// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HackernewsStoryRow container renders a snapshot 1`] = `<div />`;
